test/apps/iodemo/io_demo.cc

```diff
@@ -2365,6 +2385,12 @@ class DemoClient : public P2pDemoCommon {
             server_info_t& server_info   = _server_info[server_index];
             long total_completed         = 0;
             size_t total_bytes_completed = 0;
+            UcxLog *conn_log             = NULL;
+
+            if (opts().per_conn_info) {
+                conn_log = new UcxLog(server_info.conn->get_conn_log_prefix());
```

Comment:
maybe use `std::vector<UcxLog> log` to avoid extra allocations?

Reply:
used another way. hopes it also works for you.